src/cloud/deploy/static-content-deployment.md

```diff
@@ -8,7 +8,83 @@ functional_areas:
 
 Static content deployment (SCD) has a significant impact on the store deployment process that depends on how much content to generate—such as images, scripts, CSS, videos, themes, locales, and web pages—and when to generate the content. For example, the default strategy generates static content during the [deploy phase]({{ site.baseurl }}/cloud/deploy/cloud-deployment-process.html#-deploy-phase) when the site is in maintenance mode; however, this deployment strategy takes time to write the content directly to the mounted `pub/static` directory. You have several options or strategies to help you improve the deployment time depending on your needs.
 
-## Minify content
+## Optimize javascript and css content builds
+
+You can use bundling and minification to build optimized javascript and html content for static content deployment.
+
+### Bundle JavaScript files
+
+JavaScript bundling is an optimization technique you can use to reduce the number of server requests for JavaScript files.
+
+For {{site.data.var.ece}} projects, you can use the Magento [Baler](https://github.com/magento/baler) extension to scan generated JavaScript code and create an optimized JavaScript bundle during the build process. Deploying the optimized bundle to your site can reduce the number of network requests when loading your site and improve page load times.
+
+You must install and configure your project to use Baler.
+
+{:.procedure}
+To install and configure Baler:
+
+1. Use the following Baler extension information to [install the the Baler extension]({{ site.baseurl }}/cloud/howtos/install-components.html#install) in a {{site.var.data.ece}} development branch.
+
+   ```text
+   module name: fastly/magento2
+   repository: https://github.com/magento/m2-baler
+   ```
+
+1. In the same environment, update the `config.php` project configuration file with the following settings to enable the javascript bundling options required to use Baler.
+
+   > `config.php`
+
+   ```php
+       'js' => [
+         'merge_files' => '0',
+         'minify_files' => '0',
+         'enable_js_bundling' => '0',
+         'enable_baler_js_bundling' => '1',
+   ```
```

Comment:
There should probably be a bit more context for this (also we typically use 4 spaces to indent):

```php
'system' => [
    'default' => [
        'dev' => [
            'js' => [
                'merge_files' => '0',
                'minify_files' => '0',
                'enable_js_bundling' => '0',
                'enable_baler_js_bundling' => '1',
            ],
        ],
    ],
],
```